src/hcupc_spmd_distWS_common.cpp

```diff
@@ -289,17 +289,18 @@ void initialize_distws_setOfThieves() {
 		else {
 			printf(">>> HCLIB_DIST_WS_BASELINE\t\t= false\n");
 		}
-		if(totalTasksStolenInOneShot < 1) {
+		if(total_tasks_stolen_in_one_shot < 1) {
 			printf("WARNING (HCLIB_STEAL_N): N should always be positive integer, setting it as 1\n");
-			totalTasksStolenInOneShot = 1;
+			total_tasks_stolen_in_one_shot = 1;
 		}
-		printf(">>> HCLIB_STEAL_N\t\t= %f\n",totalTasksStolenInOneShot);
+		printf(">>> HCLIB_STEAL_N\t\t= %f\n",total_tasks_stolen_in_one_shot);
 		printf(">>> %s\n",vsdescript());
 		cout << "----------------------------------------" << endl;
 #endif
 		hclib::display_runtime();
 	}
-	assert(totalTasksStolenInOneShot <= 5);
+    // TODO: is there a reason for this? should this be a performance warning rather than an assert?
+	assert(total_tasks_stolen_in_one_shot <= 5);
```

Comment:
Yes, I don't think its required. Probably it slipped into the master branch along with some other code